joblib/test/test_parallel.py

```diff
@@ -940,3 +940,35 @@ def __reduce__(self):
     with warns(DeprecationWarning):
         with raises(ValueError):
             delayed(UnpicklableCallable(), check_pickle=True)
+
+
+@with_multiprocessing
+@parametrize('backend', ['multiprocessing', 'loky'])
+def test_backend_batch_statistics_reset(backend):
+    """Test that a parallel backend correctly resets its batch statistics."""
+
+    # let's tolerate 20% error between compuration times to avoid random
+    # failures because of variable available ressources on CIs.
+    error = 0.2
+    n_jobs = 2
+    n_inputs = 500
+    task_time = 2. / n_inputs
+
+    p = Parallel(verbose=10, n_jobs=n_jobs, backend=backend)
+    start_time = time.time()
+    p(delayed(time.sleep)(task_time) for i in range(n_inputs))
+    ref_time = time.time() - start_time
+    assert p._backend._effective_batch_size == \
```

Comment:
I would put parentheses around all the asserts rather than using backslashes (here and the other ones below). I tend to always use parentheses personally, unless really not convenient